src/components/taskDetails/index.tsx

```diff
@@ -238,13 +239,17 @@ const TaskDetails: FC<Props> = ({ taskID }) => {
                         </section>
 
                         <section className={classNames.rightContainer}>
-                            <button
-                                onClick={() =>
-                                    router.push(`/progress/${taskID}?dev=true`)
-                                }
-                            >
-                                Update Progress
-                            </button>
+                            {isDevModeEnabled && (
+                                <button
+                                    onClick={() =>
+                                        router.push(
+                                            `/progress/${taskID}?dev=true`
+                                        )
+                                    }
+                                >
+                                    Update Progress
+                                </button>
+                            )}
```

Comment:
why this changes reflect in this pr

Reply:
Actually, there's another PR which is yet to be merged, once it gets merged, these changes will no more exist 👍🏻 
Here's the PR: https://github.com/Real-Dev-Squad/website-status/pull/616

Reply:
**P.S: ** Now those changes will no more reflect in this PR.